1-exercises/C-array-every/exercise.js

```diff
@@ -5,7 +5,9 @@
 let students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 let group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-let groupIsOnlyStudents; // complete this statement
+let groupIsOnlyStudents = group.every((personInGroup) => {
+  return students.some((student) => personInGroup === student);
+}); // complete this statement
```

Comment:
great use of the .sort & .every array methods here :) you could also use .includes as an alternative solution within your .every function, for example:
`return students.includes(personInGroup)`